contracts/hackatom/src/contract.rs

```diff
@@ -30,6 +30,15 @@ pub struct MigrateMsg {
     pub verifier: HumanAddr,
 }
 
+/// SystemMsg is only expose for internal sdk modules to call.
+/// This is showing how we can expose "admin" functionality than can not be called by
+/// external users or contracts, but only trusted (native/Go) code in the blockchain
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct SystemMsg {
+    pub recipient: HumanAddr,
+    pub amount: Vec<Coin>,
+}
```

Comment:
What about structuring this as an enum, such that this gets a verb like `SystemMsg::StealFunds` as well as extensibility.

Reply:
As an example sure.

I am happy to rename it, but it is designed to allow some priviledged "system" calls as exposed to external calls. Unless you have a better name, I would like to merge and then we can rename before the 0.14 release when we have a good name.

Reply:
> As an example sure.

Yeah, I think it is easier for readers to habve a one-case enum than trying to wrap their head around a struct/enum mix.

> I am happy to rename it, but it is designed to allow some priviledged "system" calls as exposed to external calls. Unless you have a better name, I would like to merge and then we can rename before the 0.14 release when we have a good name.

Agreed, let's merge as it. Will think about it.

Reply:
Sorry, I missed this. Some late comments.

Regarding naming:
Maybe indicating the priviledged nature of the call in the name would be a good idea. `Priviledged` sounds bad, but maybe `Admin`, `Administer`, or even `Root` are good alternatives.

Reply:
So we can `root` the contracts? I like it :smile:

Reply:
oO bad things showing up in the dictionary.

But looking the root/admin direction, what about `sudo`. It is short, and a verb.

Reply:
I like it. It means "superuser do", which gives us another option: `superuser`.